src/commands/index.js

```diff
@@ -1,27 +1,27 @@
 module.exports = {
```

Comment:
heh, so If we're not typing the modules out, their order doesn't matter. This PR is purely organizational for humans, so i'm not sure automating it gets us anywhere. 😁